lib/utils/validateOptions.js

```diff
@@ -65,6 +65,17 @@ function validate(opts, ruleName, complain) {
 	const possible = opts.possible;
 	const actual = opts.actual;
 	const optional = opts.optional;
+	const whitelist = [
+		'reportDescriptionlessDisables',
+		'reportInvalidScopeDisables',
+		'reportNeedlessDisables',
+	];
+
+	if (actual === false && !whitelist.includes(ruleName)) {
+		return complain(
+			`Invalid option value \`false\` for rule "${ruleName}". Are you trying to disable this rule? If so use \`null\` instead`,
```

Comment:
[suggest] `validateOptions` uses `"` consistently to embed any values into a string,

https://github.com/stylelint/stylelint/blob/0bd6976fea7fa2a07c47672de424194716d31164/lib/utils/validateOptions.js#L180-L186

so I think using `"` is better for `false` or `null`.

```suggestion
			`Invalid option value "false" for rule "${ruleName}". Are you trying to disable this rule? If so use "null" instead`,
```